src/hotspot/os/aix/os_aix.cpp

```diff
@@ -1111,6 +1111,7 @@ bool os::dll_address_to_library_name(address addr, char* buf,
 static void* dll_load_library(const char *filename, char *ebuf, int ebuflen) {
 
   log_info(os)("attempting shared library load of %s", filename);
+  printf("Loading the filename %s\n",filename);
```

Comment:
Is this just accidentally remaining debug code or do you want to protocol each dlopen on stdout?